mesonbuild/compilers/c.py

```diff
@@ -1027,4 +1027,28 @@ def get_instruction_set_args(self, instruction_set):
             return None
         return vs32_instruction_set_args.get(instruction_set, None)
 
+    def get_toolset_version(self):
+	# See boost/config/compiler/visualc.cpp for up to date mapping
+        version = int(''.join(self.version.split('.')[0:2]))
```

Comment:
__[flake8]__
* [E101] indentation contains mixed spaces and tabs

<!-- Comment by [SideCI](https://sideci.com) -->